src/main/java/uk/gov/crowncommercial/dts/scale/cat/controller/TendersController.java

```diff
@@ -132,4 +141,47 @@ public List<Release> getProjectsDeltas(
       
       return procurementEventService.getProjectUpdatesByLastUpdateDate(lastSuccessRun, jaggaerAPIConfig.getAssistedProcurementId() );
   }
+  
+  @GetMapping("/projects/rfxtemplatemapping/mappings")
+  @TrackExecutionTime
+  public ResponseEntity<List<RfxTemplateMapping>> getAllTemplateMappings(final ApiKeyAuthToken authentication) {
+    try {
+      List<RfxTemplateMapping> mappings = fileService.getAllTemplateMappings();
+      log.info("mappings: {}",mappings);
+
+      if (mappings.isEmpty()) {
+          log.info("mappings.IsEmpty() = TRUE");
+        return new ResponseEntity<>(HttpStatus.NO_CONTENT);
+      }
+
+      log.info("mappings: {}",mappings);
+
+      return new ResponseEntity<>(mappings, HttpStatus.OK);
+    } catch (Exception e) {
+      return new ResponseEntity<>(null, HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+  }
+  
+  @PostMapping("/projects/rfxtemplatemapping/upload")
+  @TrackExecutionTime
+  public ResponseEntity<ResponseMessage> uploadFile(@RequestParam("file") MultipartFile file,final ApiKeyAuthToken authentication) {
+    String message = "";
+
+    if (CSVFileUtils.hasCSVFormat(file)) {
```

Comment:
More a personal coding style but I would invert the logic `!(CSVFileUtils.hasCSVFormat(file)` and bail out early/fail fast as it keeps the fail response code and return values next to the condition.